src/docs/pages/component-docs/components/drawer/drawer-header/examples/with-custom-content.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+
+export const WITH_CUSTOM_CONTENT = `<blui-drawer style="width: 250px">
+    <blui-drawer-header>
+        <div blui-title-content style="display: flex; align-items: start; flex-direction: column">
+               <div class="mat-h4" style="margin-bottom: -8px; margin-top: 8px">Customizable</div>
+               <div class="mat-h2" style="margin: 0">Header Content</div>
+        </div>
```

Comment:
For the sake of my terminal OCD...

```suggestion
        <div blui-title-content style="display: flex; justify-content: center; flex-direction: column">
               <div class="mat-h4" style="margin-bottom: -8px;">Customizable</div>
               <div class="mat-h2" style="margin: 0">Header Content</div>
        </div>
```

Reply:
All good, it's updated now & will push it out in a bit